crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,305 @@
+function CrowdsourceHinter(runtime, element, data){
+
+    var executeHinter = true;
+    var isShowingHintFeedback = false;
+    var voted = false;
+    var correctSubmission = false;
```

Comment:
Advanced hint: You may consider keeping unused functionality off of master, and in a branch.